merkletree/src/internal.rs

```diff
@@ -14,7 +14,7 @@
 // limitations under the License.
 
 // TODO: consider removing this in the future when fixed-hash fixes this problem
-#![allow(clippy::incorrect_clone_impl_on_copy_type)]
+#![allow(clippy::non_canonical_clone_impl)]
```

Comment:
Is the TODO logged as an issue?

Reply:
Not sure I understand what you mean. This is a minor issue that can be casually checked every now and then. Nothing that we care about.